app/views/submissions/show.erb

```diff
@@ -13,8 +13,24 @@
         <%= erb :"submissions/user_actions", locals: { submission: submission, exercise: submission.user_exercise } %>
 
         <% if current_user.owns?(submission) %>
+          <%
+            twitt_params =
```

Comment:
I'm actually comfortable with params if it's for a route, but in this context I think we could come up with a better name. I'm just not sure what is actually in this yet :)